virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -165,6 +161,7 @@ Deprecated features are included in the current release and supported. However,
 * The `copy-template`, `modify-vm-template`, and `create-vm-from-template` tasks are deprecated.
 
 //CNV-29048 Release note: Metrics name changes
+//Check if there is a 4.15 article that we can link to
 * Many OpenShift Virtualization metrics have changed or will change in a future version. These changes could affect your custom dashboards. See link:https://access.redhat.com/articles/7028805[OpenShift Virtualization 4.14 metric changes] for details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2179660[*BZ#2179660*])
```

Comment:
Do we need a similar note as above for 4.15 metric changes? If so, can I please get a link to the relevant article?

Reply:
@kmajcher-rh @sradco Can you please provide your input?

Reply:
I have commented this note out for 4.15